nireports/tests/test_reportlets.py

```diff
@@ -486,3 +488,24 @@ def test_plot_raincloud(orient, density, tmp_path):
         nans_value=nans_value,
         output_file=output_file,
     )
+
+
+def test_plot_segs(tmp_path, request):
```

Comment:
The fixture `outdir` is None if you want to write to a temporal folder and forget, but when it points to a path, then the results should be stored them and can be visualized (e.g., on circle)

```suggestion
def test_plot_segs(request, outdir):
```